.github/java-config.yml

```diff
@@ -0,0 +1,2 @@
+java-version: "17"
+distribution: "temurin"
```

Comment:
There is not a builtin vanilla way for gh actions workflows to read a file like this right? So we need a third party action

Reply:
> There is not a builtin vanilla way for gh actions workflows to read a file like this right? So we need a third party action

Unfortunately not (or at least not that I'm aware of).